### DISCUSSION TITLE
slingbox reset

### DISCUSSION THREAD

**Discussion by ceenicol** [GerryDazoo/Slinger#167] [Q&A]
Is a reset on the slingbox unit required?  It seems your code is functioning..as far as i can tell, self discovery finds the slingbox, but it shows the original name, and correct ip and port :5319 had been assigned using Upnp, which i disabled on the router.   I'd like to get this working utilizing the python version. Any suggestions?
Thanks in advance, your work on this project is incredible.
regards,colin

/tmp/sling.log after fresh start of sling.service
        note: config.ini lets your code find the slingbox.  Long Corner Slingbox is the original name of my slingbox??

Version : 3.08 Running on Linux-5.15.61-v7+-armv7l-with-glibc2.31 pid= 532
Using config file config.ini
Connection Manager Running on port 8080 with 10 max streams using URL slingbox.
Streamer Running:  starting up on  port 808010  8388608config.ini
 SLINGBOX 8080 8080 8388608
8080 No valid slingbox ip info found in config.ini
Finding Slingbox on local network. My IP Info =  b8:27:eb:c3:66:58
Finding Slingbox on local network. My IP Info =  192.168.1.20
Slingbox Found 192.168.1.158 5319 " Long Corner Slingbox^@^@^@^@^@^@^@^@^@^@^@^@ " FinderID D08B5D0448A3C6E98ABD989
8080 Using slingbox at  ('192.168.1.158', 5319)
Streamer:  8080 Waiting for first stream, flushing any IR requests that arrive while not connected to>

after atttempting to connect from VLC

Hacker Alert. Invalid Request from  ('192.168.1.193', 48420)
GET /Slingbox HTTP/1.1
Host: 192.168.1.20:8080
Accept: */*
Accept-Language: en_US
User-Agent: VLC/3.0.16 LibVLC/3.0.16
Range: bytes=0-

Hacker Alert. Invalid Request from  ('192.168.1.193', 48436)
GET /Slingbox HTTP/1.0
Host: 192.168.1.20:8080
User-Agent: VLC/3.0.16 LibVLC/3.0.16
Icy-MetaData: 1

slingbox config.ini

[SLINGBOX]
sbtype="350/500/M1/M2"
;sbtype="Solo/Pro/ProHD"
;sbtype="240"
password=b8m1x3y3zbv4p31
; uncomment the next lines ipaddress and port with your local network info
; if you don't want the server to automatically find your 
; slingbox on the local network. Needed if server and slingbox are
; not on the same LAN segment. Also removes requirement for the netifaces module  
;ipaddress=192.168.1.158
;port=5302
; Repace with your finderid, If and only if you need remote access and don't  
;have a static ip and you plan on using my service to access you server remotely. 
; Please read the release notes. Most people don't need this.
;finderid=BAD95DAED609364C8204F2BC5FBDC363

;If you've got more than one slingbox set name. THe server will use this when generating 
; logs to make it easier to see what's going on. 
;name=something

; Following are the default values
; Default resolution when server starts. You can change this and reconnect without having to restart the server
; valid range = 1..15
Resolution=12

; Valid Values 1, 6, 10, 15, 20, 30, 60   depending on resolution
FrameRate=30

;Min 50 Max 8000
VideoBandwidth=2000

; Min 0 (auto) max 63
VideoSmoothness=63

; Send Iframe every n seconds Max 30
IframeRate=5

; Audio Bit Rate. Valid Options 16, 20, 32, 40, 48, 64, 96 Default 32
AudioBitRate=32

; Video Source  0, 1, 2, 3 depending on your hardware corrosponds to one of
;Composite, Component, HDMI or Tuner.
;ProHD 0=Tuner, 1=composite 2=svideo 3=component
;Solo, M1 0=composite 1=Svideo 2=component
;500  0=composite, 1=component, 2=HDMI
; If you don't set this the code will use the last configired input
; I recommend setting this value. If not set correctly you'll often only see a black screen. 
VideoSource = 2

[SERVER]
; local port number for the server to listen on for connections
port=8080
;maxstreams=10
enableremote=no

[REMOTE]
; see release notes for changing this if your remote doesn't work, But as a start make it 
;the same as your configured VideoSource 0-3. See above.
code=1
; path to an external remote control definitions file, if the defaults
; aren't working for you or you want to change the format of the web page.
; use the supplied remote.txt as a starting point.
;include=remote.txt

**Comment by GerryDazoo** [comment]
please use the documented URLs   /slingbox NOT Slingbox.
And just in case make sure you use /Remote not /remote for th web remote control.
